src/auth/Fingerprint.hpp

```diff
@@ -27,6 +27,8 @@ class CFingerprint : public IAuthImplementation {
   private:
     struct SDBUSState {
         std::string                         message = "";
+        std::string                         failure_reason = "";
```

Comment:
`failureReason`

Reply:
oh I also forgot to delete `message` after refactoring

Reply:
Actually while your at it, can you move the strings out of `SDBUSState`?
Either just `m_sMessage, m_sFailureState` and co, or in their own struct. Feel like they don't belong there.

Reply:
sure that's fair